docs/kind.md

```diff
@@ -1,22 +1,86 @@
 # OVN kubernetes KIND Setup
 
-KIND deployment of OVN kubernetes is a fast and easy means to quickly install and test kubernetes with OVN kubernetes CNI.  The value proposition is really for developers who want to reproduce an issue or test a fix in an environment that can be brought up locally and within a few minutes.
+KIND (Kubernetes in Docker) deployment of OVN kubernetes is a fast and easy means to quickly install and test kubernetes with OVN kubernetes CNI.  The value proposition is really for developers who want to reproduce an issue or test a fix in an environment that can be brought up locally and within a few minutes.
 
-### Prerequisites
+### Prerequisites 
 
 - 20 GB of free space in root file system
 - Docker run time
-- KIND  ( https://kubernetes.io/docs/setup/learning-environment/kind/ )
-- kubectl ( https://kubernetes.io/docs/tasks/tools/install-kubectl/ )
+- [KIND]( https://kubernetes.io/docs/setup/learning-environment/kind/ )
+   - Installation instructions can be found at https://github.com/kubernetes-sigs/kind#installation-and-usage. 
+   - NOTE: The OVN-Kubernetes [ovn-kubernetes/contrib/kind.sh](https://github.com/ovn-org/ovn-kubernetes/blob/master/contrib/kind.sh) and [ovn-kubernetes/contrib/kind.yaml](https://github.com/ovn-org/ovn-kubernetes/blob/master/contrib/kind.yaml) files provision port 11337. If firewalld is enabled, this port will need to be unblocked:
```

Comment:
I think 11337 is no longer the default port with changes @aojea recently made.

Reply:
It does, this port is hardcoded 
https://github.com/ovn-org/ovn-kubernetes/blob/60427cea5883428df1223df073de162fb05abe77/contrib/kind.yaml.j2#L7

Reply:
Sounds good I will leave it as is here